spec/forms/backup_code_setup_form_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+describe BackupCodeSetupForm do
+  it 'returns the correct domain' do
```

Comment:
Why do we care what domain the setup form returns? We don't use the domain_name method anywhere in the application.